README.md

```diff
@@ -320,14 +320,18 @@ The generator update enables Open Interpreter to be controlled via HTTP REST end
 ```python
 # server.py
 
-from fastapi import FastAPI, Response
+from fastapi import FastAPI
+from fastapi.responses import StreamingResponse
 import interpreter
 
 app = FastAPI()
 
 @app.get("/chat")
-def chat_endpoint(message):
-    return Response(interpreter.chat(message, stream=True), media_type="text/event-stream")
+def chat_endpoint(message: str):
+    def event_stream():
+        for result in interpreter.chat(message, stream=True):
+            yield f"data: {result}\n\n"
+    return StreamingResponse(event_stream(), media_type="text/event-stream")
```

Comment:
This is a minor nitpick, but can we put a line break before this `return` so it’s easier for the reader to scan and parse the whole handler? Or maybe move the `event_stream` function declaration out of the `chat_endpoint` handler?